integrationTests/consensus/testInitializer.go

```diff
@@ -91,7 +91,7 @@ func genValidatorsFromPubKeys(pubKeysMap map[uint32][]string) map[uint32][]shard
 	for shardId, shardNodesPks := range pubKeysMap {
 		shardValidators := make([]sharding.Validator, 0)
 		for i := 0; i < len(shardNodesPks); i++ {
-			v, _ := sharding.NewValidator(big.NewInt(0), 1, []byte(shardNodesPks[i]))
+			v, _ := sharding.NewValidator(big.NewInt(0), 1, []byte(shardNodesPks[i]), []byte(shardNodesPks[i]))
```

Comment:
BLS pubkeys are not the same length as schnorr.

Reply:
Yes, this is a dummy address anyway. Changed to be at least different compared to pubkey parameter.